packages/backend/src/nest/ipfs/ipfs.service.ts

```diff
@@ -107,14 +122,21 @@ export class IpfsService {
       throw new Error(`Datastore keyEncoding was set to ${datastoreInit.keyEncoding} but MUST be set to 'utf8'!`)
     }
 
-    const datastoreLevelDb = new Level<string, Uint8Array>(this.ipfsRepoPath + '/data', datastoreInit)
+    const basePath = repoPathOverride || this.ipfsRepoPath
+    const dataDir = path.join(basePath, `data_${this.auth.team.id ?? ''}`)
```

Comment:
Is the team.id sufficiently collision resistant that we'll never see collisions?

Also what if someone leaves and rejoins the same team?

Reply:
Also is everything deleted on leaving a community? What files remain?